### ISSUE TITLE
Design: call to action on details pages

### ISSUE THREAD

**Issue by MatMoore** [ministryofjustice/find-moj-data#951]
## Problem
On chart pages, the main call to action is "access this data", which appears below the chart details.

However, there is also "request access" in the sidebar. These two actions are related, and if the chart is public, then the request access one is redundant.

At the moment, all charts are public, and display some default text under "request access"

![Screenshot showing "access this data" in the main content area and "request access" in the sidebar](https://github.com/user-attachments/assets/df331e21-f05f-4dce-b375-64be018662c2)

## Questions
Can we merge these into a single "Access this data" section?

This would mean that these related actions appear in the same place in the layout, and if the thing is public, we just show the link.

## Other considerations

At some point we will have some kind of "open in analaytical platform" button for tables/databases as well. The pattern we use for that should be consistent with what we do for charts.

**Comment by MatMoore** [comment]
Yeah I think the design should be consistent across all entity types.

**Comment by MatMoore** [comment]
@joe-horton-moj I've had a look but and left some comments, but I don't think I know enough about the process for gaining access, not having done it myself - maybe @LavMatt and @murdo-moj could take a look, and we could potentially show it to AP team as well? I know this process will be redesigned at some point, but in the meantime I think we could still clarify the steps for people who aren't on the AP already.

**Comment by joe-horton-moj** [comment]
Having a chat with AP team on Thursday 6th so will update designs based off that chat 👍

**Comment by joe-horton-moj** [comment]
### Logic

The text that will be shown depends on the values of `security_classification`, `access_requirements`, and `is_access_url`. 

**Here’s how it works:**
If `security_classification` is "Official-Sensitive", the "Request access" section will be displayed. Otherwise, nothing appears.

**If `access_requirements` exists:**

If `is_access_url` is true, it will show:
> Select this link for access information (opens in new tab)

If `is_access_url` is false and `data_custodians` or `data_owner` is true: 
> Contact the data custodian to request access.

**If `access_requirements` does not exist:**
> Not provided

### Problem

1. Select this link for access information is quite a leap for a non-technical user
2. The AP guidance isn't as clear as it could be to help get set-up and access this data in AP
3. Whether `is_access_url`is true or false, is the process to access the data still not the same (pull request)?

Problem 1 and 2 could be improved from @sdurnberger work on iterating the user guide.

### Proposed solutions

Problem 3 can be solved in the data catalogue. I want to double check what the process is for accessing CaDeT data in the AP.

**Access the data in the Analytical Platform**

You need to do X. Read more about this on the user guide. 

You need to ask the technical contact to review your pull request to access the data.

**Comment by joe-horton-moj** [comment]
Questions I have:

1. Whether there is a data custodian for the entity / dataset or not, is the process the same to access the data in AP?
2. What are the different ways in which you can request access to view data in the AP?
3. Will this section of the AP user guide be improved to make it clearer to less technical users? **Yes – chat with Sabrina**

**Comment by joe-horton-moj** [comment]
### Solution

Had a chat with Sabrina – she is working on the AP user guide and specifically, the opening data in AP section. We decided it made sense to just improve the content of the link in the data catalogue as improvements to the AP user guide will be made shortly. We'll keep an eye on this particular interaction though in user research to see if it's a steep drop off for non-technical users who aren't aware of the AP.

**Content change:**
- H2 is `Access this data`
- Link to change to `Select this link for access information (opens in new tab)`
- padding/margin-7 (40px) to top and bottom of `Access this data` section

<img width="1552" alt="Image" src="https://github.com/user-attachments/assets/3b6a270e-be1d-49f8-9965-77006e20d90f" />

**Comment by murdo-moj** [comment]
https://github.com/ministryofjustice/find-moj-data/pull/1425
